cypress/e2e/awx/administration/workflowApproval.cy.ts

```diff
@@ -7,18 +8,255 @@ import { User } from '../../../../frontend/awx/interfaces/User';
+                  workflowApprovalList.push(wFApprov);
+                  // FIXME: find a way to retrieve and assert all IDs reliably
+                  // workflowApprovalListIDs.push(wFApprov.id);
+                });
+              });
+            });
+        }
+      });
+
+      it('admin can enable concurrent jobs in a WFJT with a WF approval node, launch the job multiple times, bulk approve, and then bulk delete from the list toolbar', () => {
+        bulkActAssertAndDeleteWorkflowApproval('approve');
+      });
+
+      it('admin can enable concurrent jobs in a WFJT with a WF approval node, launch the job multiple times, bulk deny, and then bulk delete from the list toolbar', () => {
+        bulkActAssertAndDeleteWorkflowApproval('deny');
+      });
+    });
+  });
+
+  describe('Detail Screen', () => {
+    before('', () => {});
+    // Workflow Approval Detail Screen (not yet implemented in the new UI):
+    // User can approve a workflow approval from the details screen
+    // User can deny a workflow approval from the details screen
+    // User can cancel a workflow approval from the details screen
+    // User can approve or deny a workflow approval from the list view and then visit the details page of the workflow approval to delete it.
+  });
+
+  describe('Job Output Screen', () => {
+    before('', () => {});
+    // Workflow Approval Job Output Screen (not yet implemented in the new UI):
+    // User can access the output screen of a running workflow job template with a workflow approval node, access the workflow approval details page by clicking on the node in the output screen, and approve the workflow approval.
+    // User can access the output screen of a running workflow job template with a workflow approval node, access the workflow approval details page by clicking on the node in the output screen, and deny the workflow approval.
+    // User can deny a workflow approval, visit the output screen of the workflow approval job, click on the node of the failed job, and see that the reason for the failure is that the workflow approval was denied.
+    // User can visit the workflow visualizer of a workflow job template from the workflow approval details screen.
+  });
+
+  describe('Job Details Screen', () => {
+    before('', () => {});
+    // Workflow Approval Job Details Screen (not yet implemented in the new UI):
+    // User can relaunch the workflow from the workflow approval job details screen.
+    // User can cancel the workflow from the workflow approval job details screen.
+  });
+});
 
 /* Shared functions across test cases */
-const wfaURL = '/administration/workflow-approvals/';
+
 function actAssertAndDeleteWorkflowApproval(
   selectorDataCy: 'approve' | 'deny' | 'cancel',
-  wfaID: string
+  wFApprovID: string
 ) {
   let actionStatusCode: number;
   let statusText: string;
   let actionURL: string;
 
-  cy.visit(wfaURL);
+  cy.visit(wFApprovURL);
```

Comment:
@fullsushidev  any reason for `cy.visit` and not `navigate`

Reply:
`cy.navigateTo('awx', 'workflow-approvals')` should do the same as `cy.visit` @fullsushidev

Reply:
It was to make it easier since this piece of code will repeat a lot in this spec, although there's only 2 uses now.

Reply:
Hard coding the urls is going to make it hard for us to get the upstream tests working against the downstream platform when we get to that point as the urls change in the platform. using cy.navigate will handle this properly for us.